### ISSUE TITLE
Can't reproduce the result

### ISSUE THREAD

**Issue by ToughLife** [YooJiHyeong/InvertibleGrayscale_PyTorch#1]
Hi, much appreciate for your work!
I have run your code on Ubuntu 18.04, Pytorch 1.5.0, 4 GPUS, with 13,758 images from PASCAL VOC 2012 as training data (just as mentioned in the paper). 
But the results are very bad, the colorized image is of great different from the original one.
Since you do not provide the `train_v1.csv` and `test_v1.csv`, I just shuffle the image name in PASCAL VOC 2012 and write it to `train_v1.csv`
Have you test your code with PASCAL VOC ? what about the result?
Thanks!
